mrbgems/mruby-eval/src/eval.c

```diff
@@ -298,7 +298,7 @@ f_eval(mrb_state *mrb, mrb_value self)
 
   mrb_get_args(mrb, "s|ozi", &s, &len, &binding, &file, &line);
 
-  proc = create_proc_from_string(mrb, s, len, binding, file, line);
+  proc = create_proc_from_string(mrb, s, (int)len, binding, file, line);
```

Comment:
This cast means nothing because `len` is casted to `size_t` in `create_proc_from_string`. I'd rather change the prototype of `create_proc_from_string` to take `mrb_int len`.

Reply:
Fixed.